packages/terra-alert/src/_variables.scss

```diff
@@ -0,0 +1,71 @@
+$terra-alert-alert-color-scheme: (
```

Comment:
Sorry, I'm not clear on what you are asking for.  Can you point be to an example in one of the other components?